src/nvim/file_search.c

```diff
@@ -1372,6 +1373,12 @@ char *find_file_in_path_option(char *ptr, size_t len, int options, int first, ch
           memmove(ptr, ptr + 1, strlen(ptr));
         }
       }
+      path = xstrdup(path_option);
+      for (ptr = path; *ptr != NUL; ptr++) {
```

Comment:
`path` is never freed (that's why asan ci is failing).

Reply:
Thanks!

I'm setting up ASAN locally to try the fix, just using `xfree` raised another error. Probably because I we reference that value on a static pointer (a wild guess from a non-C developer).